tests/Diagnostics.Tests.Integration.RuntimeHost/IntegrationTests.cs

```diff
@@ -30,5 +33,56 @@ public async Task HealthPingTest(string url)
             Assert.Equal("application/json; charset=utf-8",
                 response.Content.Headers.ContentType.ToString());
         }
+
+        [Theory]
+        [InlineData("/subscriptions/ef90e930-9d7f-4a60-8a99-748e0eea69de/resourceGroups/detector-development/providers/Microsoft.Web/sites/buggybakery/detectors")]
```

Comment:
Is it possible to feed this data via config instead of code?

Reply:
Check this out: https://andrewlock.net/creating-a-custom-xunit-theory-test-dataattribute-to-load-data-from-json-files/

Reply:
You can imagine using the same method to test List Detectors for multiple resource types by changing the resource and minimum expected detector count.
- Web App
- Function App
- Logic App
- AKS